lib/do_username.rb

```diff
@@ -1,12 +1,12 @@
 module DOUsername
-  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray giantSquid shrimp mantaRay angler snorkler scubaDiver).freeze
-  SEA_OBJECTS = %w(boat ship submarine yacht dinghy raft kelp).freeze
+  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray giantSquid shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl).freeze
```

Comment:
```suggestion
  SEA_CREATURES = %w(walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl).freeze
```
As giant is being added to the size descriptors, we can just remove the giant variant of squid here

Reply:
No more GiantGiantSquid 😛